.travis.yml

```diff
@@ -12,6 +12,7 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
+  - "3.7"
```

Comment:
@surajbarkale I'm afraid Travis does not support Python  3.7 yet.
Refer to https://github.com/travis-ci/travis-ci/issues/9815